bridget/eddy/footprint.py

```diff
@@ -277,71 +315,103 @@ def _footprint_korman(t_air, a, p, u, cov_uw, cov_vw, cov_wt, var_v, z = 2.0):
         absolute humidity [g/m3].
     p : list
         air pressure [hPa].
+    direction : list
+        wind direction [°].
+    time : list
+        datetime list corresponding to the other variables, format: '%m.%d.%Y %H:%M'.
```

Comment:
We should accept time only as  `datetime` objects. Transforming input data into the correct data types in python is the responsibility of the user.

Reply:
But wouldn't that be the case for all of the input parameters?
Should the data type be a list everywhere or should it be float?
The function should expect lists of floats for most of the variables.

Reply:
Yeah, to clarify: the time parameter should accept a list of datetime objects.

Reply:
I mean, there are two avenues:

accept lists or accept only the literal values. You almost implemented a solution of both, which is convenient and fine. 
See also [my comment above](https://github.com/KIT-HYD/bridget/pull/15#discussion_r539084421). If a tstamp index is given, calculate for the given index, else create an index and call the function recursively.

Reply:
I see what you mean, but how do i implement this in the docstring?

1)   `time : datetime`
2)   `time : list`

At the moment, the function would even work without giving a list of the measurement times, as it is never used and the user gives the index of the "line" to calculate with the tstamp parameter (I changed this in the last days).
But I think the measurement time is just an essential parameter and could be needed for further development.